docs.bruno/Testing - get data.bru

```diff
@@ -0,0 +1,11 @@
+meta {
+  name: Testing - get data
+  type: http
+  seq: 1
+}
+
+get {
+  url: https://api.github.com/users/usebruno
```

Comment:
Consider adding error handling for the GET request. This could include handling for network errors, server errors, or unexpected response formats. [important]